website/server/controllers/api-v3/debug.js

```diff
@@ -204,6 +208,41 @@ api.questProgress = {
   },
 };
 
+/**
+ * @api {post} /api/v3/debug/boss-rage Artificially trigger boss rage bar
+ * @apiName bossRage
+ * @apiGroup Development
+ * @apiPermission Developers
+ *
+ * @apiSuccess {Object} data An empty Object
+ */
+
+api.bossRage = {
```

Comment:
This should probably have some tests. At least those that check that the call is correctly working/removed when in debug mode vs not.

Reply:
I'll work on adding those. Thanks for the catch!